packages/main/src/system/windows-startup.ts

```diff
@@ -87,12 +102,16 @@ export class WindowsStartup {
     // write the script file
     // also make sure we try to execute the binary only if it exists to avoid any start error/annoying popup
     // in case user removed the app
-    const vbsContent = `' This file is automatically generated from Podman Desktop preferences.
+    // Use """" instead of chr(34) escape codes in the script to avoid any issue with the path / adding --minimized
+    const vbsContent =
+      `' This file is automatically generated from Podman Desktop preferences.
 set FSO = CreateObject("Scripting.FileSystemObject")
 Set WshShell = CreateObject("WScript.Shell")
 if FSO.FileExists("${this.podmanDesktopBinaryPath}") then
   WshShell.CurrentDirectory = "${this.exeDirectory}"
-  WshShell.Run chr(34) & "${this.podmanDesktopBinaryPath}" & Chr(34), 0
```

Comment:
fyi @benoitf we use `"""` for the script instead as it's cleaner than multiple `Chr(34)` which is double-quotes anyways.